doc/040_backup.rst

```diff
@@ -366,6 +366,12 @@ created as it would only be written at the very (successful) end of
 the backup operation.  Previous snapshots will still be there and will still
 work.
 
+Errors
+*********************
+
+When running a backup, restic displays the number of errors that occured while creating the backup.
```

Comment:
Need to mention critical errors vs source read errors. "Snapshot will be completed even if some source data could not be read correctly. Critical errors such as failure to write to the backup destination not generate a snapshot."